src/SFA.DAS.RoATPService.Application.Api/Controllers/FatDataExportController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.RoATPService.Application.Api.Middleware;
+using SFA.DAS.RoATPService.Application.Interfaces;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace SFA.DAS.RoATPService.Application.Api.Controllers
+{
+    [ApiController]
+    [Authorize(Roles = "RoATPServiceInternalAPI")]
+    [Route("api/v1/fat-data-export")]
+    public class FatDataExportController : ControllerBase
+    {
+        private readonly ILogger<FatDataExportController> _logger;
+        private readonly IFatDataExportRepository _repository;
+
+        public FatDataExportController(ILogger<FatDataExportController> logger,
+            IFatDataExportRepository repository)
+        {
+            _logger = logger;
+            _repository = repository;
+        }
+
+        [HttpGet("")]
```

Comment:
optional, but I'd add a route here to allow to to expand logically if you ever want more data exports. Something to describe your requirements better,  and maybe rename the sproc to also reflect that? All optional though.

Reply:
If we are going to expand it would be in another controller action

Reply:
left it as the default route for now, which is defined at controller level.